vhs-frontend/src/components/searchBarConfig.ts

```diff
@@ -0,0 +1,9 @@
+export type SearchFormInput = {
+  queryType: string;
```

Comment:
Not strictly necessary, but use zod for validation even if the whole schema is optional so that you get used to it.

Reply:
Don't forget one of the above comments to make `queryType` a union type.